### DISCUSSION TITLE
为什么InterestingDarkness黑暗佬的ChatSydney从昨天晚上开始我这边打开闪一下聊天记录就白屏了没反应啊。

### DISCUSSION THREAD

**Discussion by pointm** [viopsa233/AI-testers-Forum#62] [Q&A]
![image](https://github.com/viopsa233/AI-testers-Forum/assets/49180699/722e6500-8f5d-466a-8dcc-479304844e5b)
如图，就是这个样子的白屏。
![image](https://github.com/viopsa233/AI-testers-Forum/assets/49180699/a334a086-2fe5-44f1-9450-930d425252f3)
min命令行这边应该是正常的。
试着重新删掉克隆了一下再安装库发现说没有ChatEdge0.4.4版本，是我网络出问题了吗？

**Comment by ghost** [comment]
pip不要用第三方源

**Reply by pointm** [reply]
> pip不要用第三方源

好吧，我现在更新了一下EDGEGPT，但是点开链接之后还是显示白屏（悲

**Reply by ghost** [reply]
白屏是JS没加载完，最好翻墙

**Reply by pointm** [reply]
> 白屏是JS没加载完，最好翻墙

好的，我试试，谢谢暗黑佬

**Reply by pointm** [reply]
> 白屏是JS没加载完，最好翻墙

没事了佬，我更新完了之后，清了一下浏览器缓存就好了
